### PR TITLE
[Storage] add warning to inform default storage account change in future

### PR DESCRIPTION
**Pull request by Juliehzl**
---
Fix  https://msazure.visualstudio.com/One/_workitems/edit/4909160. Because default storage account change is a breaking change, announce a warning first and change in a month. 

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/storage/operations/account.py [hunk 1]
```diff
@@ -18,6 +20,7 @@ def create_storage_account(cmd, resource_group_name, account_name, sku=None, loc
         cmd.get_models('StorageAccountCreateParameters', 'Kind', 'Sku', 'CustomDomain', 'AccessTier', 'Identity',
                        'Encryption', 'NetworkRuleSet')
     scf = storage_client_factory(cmd.cli_ctx)
+    logger.warning("The default kind for created storage account will change to 'StorageV2' from 'Storage' in future")
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/storage/operations/account.py]: Does kind mean SKU?

Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/storage/operations/account.py]: NO. It is for storage account kind.

**Comment by cg6** [comment]
This commit is a breaking change. A 100% successful storage account create now throws a warning.
Unfortunately if calling az from Powershell it interprets this as an error and breaks the script.

PS C:\> $Error[0]
The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: WARNING: The default kind for created storage account will change to 'StorageV2' from 'Storage' in future

**Comment by MyronFanQiu** [comment]
@Juliehzl

**Comment by Juliehzl** [comment]
> This commit is a breaking change. A 100% successful storage account create now throws a warning.
> Unfortunately if calling az from Powershell it interprets this as an error and breaks the script.
> 
> PS C:> $Error[0]
> The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: WARNING: The default kind for created storage account will change to 'StorageV2' from 'Storage' in future

Hi @cg6, it works for me as follows in PowerShell. Can you give me more information about your error scenario?
```
PS C:\Users\zuh\Desktop\test_folder> az storage account create -g zuh -n zuhstoragetest4
The default kind for created storage account will change to 'StorageV2' from 'Storage' in future
{
  "accessTier": null,
  "azureFilesIdentityBasedAuthentication": null,
  "creationTime": "2019-09-24T02:59:53.281508+00:00",
  "customDomain": null,
  "enableHttpsTrafficOnly": true,
  "encryption": {
    "keySource": "Microsoft.Storage",
    "keyVaultProperties": null,
    "services": {
      "blob": {
        "enabled": true,
        "lastEnabledTime": "2019-09-24T02:59:53.359615+00:00"
      },
      "file": {
        "enabled": true,
        "lastEnabledTime": "2019-09-24T02:59:53.359615+00:00"
      },
      "queue": null,
      "table": null
    }
  },
...
```

**Comment by yugangw-msft** [comment]
Quite a few CLI commands emit warnings for important information. @cg6, are you in PowerShell ISE which would stop execution on detecting anything from stderr?

**Comment by cg6** [comment]
@yugangw-msft you are correct. I had an unrelated issue and was stepping through in ISE. Outside of the ISE it's fine.
Apologies.

**Pull request merged**